azure/table_azure_subnet.go

```diff
@@ -311,6 +311,9 @@ func getIpConfiguration(ctx context.Context, ipConfig *network.IPConfiguration,
 
 	configuration, err := client.Get(ctx, resourceGroup, networkInterface, configName)
 	if err != nil {
+		if strings.Contains(err.Error(), "ResourceNotFound") {
+			return nil, nil
+		}
```

Comment:
@ParthaI Can we use the `isNotFoundError` function mentioned in https://github.com/turbot/steampipe-plugin-azure/blob/main/azure/errors.go#L11 to handle this error?

Reply:
@misraved this is a separate hydrate call, not a part of the get/list config. So we have to handle exceptions in this way.

Reply:
Thanks for the explanation @bigdatasourav 👍.